src/index.ts

```diff
@@ -100,6 +112,24 @@ export class PaymentSocket extends EventEmitter {
     return this._totalDelivered.toString()
   }
 
+  get exchangeRate (): string {
+    if (!this.connected) {
+      throw new Error('Must be connected to get exchangeRate')
```

Comment:
Just a nitpick, but I think it's confusing for a getter to throw an error, since from the user's perspective they are just accessing a property.